dev/merge_spark_pr.py

```diff
@@ -44,7 +44,7 @@
 
 # Location of your Spark git development area
 SPARK_HOME = os.environ.get("SPARK_HOME", os.getcwd())
-# Remote name which points to the Gihub site
+# Remote name which points to the Github site
```

Comment:
This is correct, but should it be in an independent pr?

Reply:
@LuciferYang ok, let me send a separate minor PR for it.

Reply:
https://github.com/apache/spark/pull/42032